modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -415,6 +420,9 @@ class Session implements ISession {
         this.statsEnabled = observers.any { it.enableMetrics() }
         this.workflowMetadata = new WorkflowMetadata(this, scriptFile)
 
+        // download custom fsync binary which supports fsyncing of every file in a directory
+        this.downloadCustomFsync()
```

Comment:
can we just put this in the Nextflow base image at /root and then copy it in from there? avoids having to download and deal with download failures